### PR TITLE
(maint) Update GPG key

### PR DESCRIPTION
**Pull request by mwaggett**

### CODE REVIEW

File: ext/build_defaults.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ project: 'puppetlabs-release-pc1'
 packaging_url: 'git://github.com/puppetlabs/packaging.git --branch=master'
 packaging_repo: 'packaging'
 gpg_name: 'info@puppetlabs.com'
-gpg_key: '4BD6EC30'
+gpg_key: 'EF8D349F'
```
Comment (underscorgan) [inline ext/build_defaults.yaml]: Can we use the long key instead of the short key here?

Comment (mwaggett) [inline ext/build_defaults.yaml]: Sure, if you give it to me. What's the difference?

Comment (underscorgan) [inline ext/build_defaults.yaml]: `7F438280EF8D349F`

It's more secure against collisions

Comment (mwaggett) [inline ext/build_defaults.yaml]: @underscorgan Done.

File: ext/build_defaults.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ project: 'puppetlabs-release-pc1'
 packaging_url: 'git://github.com/puppetlabs/packaging.git --branch=master'
 packaging_repo: 'packaging'
 gpg_name: 'info@puppetlabs.com'
```
Comment (underscorgan) [inline ext/build_defaults.yaml]: This needs to change to release@puppet.com

Comment (mwaggett) [inline ext/build_defaults.yaml]: @underscorgan Done!

**Pull request merged**